mantidproject/mantid
# Improve HB3AFindPeaks unit test performance

**Description of work.**

Reduces the testing data set used by the unit tests for HB3AFindPeaks to improve test time and memory consumption. The test should run almost twice as fast, and will consume about 6.4GB of memory instead of 16+GB. 

**To test:**



Run the `HB3AFindPeaks` unit test to verify it runs quicker and consumes less memory.

*There is no associated issue.*


*This does not require release notes* because this is an internal unit test change.



Version into `ornl-next` #30676 

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
